README.md

```diff
@@ -74,6 +74,59 @@ fn initialize_metrics(service_log_dir: PathBuf) -> AttachHandle {
 
 You can either attach it to a global destination or thread the queue to the location you construct your metrics object directly. Currently, only formatters for [Amazon EMF] are provided, but more may be added in the future.
 
+## Glossary
```

Comment:
I was imagining this was a slightly higher level listing of terms, specifically ordered by the ones that are new to people:

- `Entry`: "an entry is the unit of work that is flushed at the end of a request or other event. An entry is typically defined by something using the `#[metric]` attribute
- `Sink`: A sink is where entries are flushed to. Builders typically configure a global sink with ...
- Guard: The standard usage pattern with an entry is to drop it use `append_on_drop` (link to example) that flushes it to a `Sink`
- (now include the fields you have below like dimension, datapoint etc.)
- `slot`: ...